docs/src/examples.rst

```diff
@@ -14,205 +14,175 @@
 |cdk| Examples
 ##############
 
-This topic contains some usage examples to help you get started understanding
-the |cdk|.
+This topic contains some examples to help you get started using some of the advanced constructs
+offered by the |cdk|.
 
-.. We'll include this if we ever implement DeploymentPipeline
-   _multiple_stacks_example:
+.. _creating_ecs_l2_example:
 
-   Creating an App with Multiple Stacks
-   ====================================
+Creating an |ECS| Construct
+===========================
 
-   The following example creates the following stacks and one deployment pipeline:
+|ECSlong| (|ECS|) is a highly scalable, fast, container management service
+that makes it easy to run, stop, and manage Docker containers on a cluster.
+You can host your cluster on a serverless infrastructure that is managed by
+|ECS| by launching your services or tasks using the Fargate launch type.
+For more control you can host your tasks on a cluster of
+|EC2long| (|EC2|) instances that you manage by using the EC2 launch type.
 
-   - **Dev** uses the default environment
-   - **PreProd** in the **us-west-2** Region
-   - **NAEast** in the **us-east-1** Region
-   - **NAWest** in the **us-west-2** Region
-   - **EU** in the **eu-west-1** Region
-   - **DeploymentPipeline** in the **us-east-1** Region
+Since |ECS| can be used with a number of AWS services,
+you should understand how the |ECS| construct that we use in this example
+gives you a leg up on using AWS services:
 
-   Implement the class **MyStack** in the *my-stack* sub-folder,
-   that extends the |stack-class| class
-   (this is the same code as shown in the :doc:`concepts` topic).
+* Automatic security group opening for load balancers
```

Comment:
These are not sentences, maybe rephrase to a list of the form "it will open up ..." etc.